cookbooks/third-party/nfs-5.0.0/kitchen.platforms.yml

```diff
@@ -21,9 +21,6 @@ platforms:
   - name: fedora-latest
     driver:
       box_url: https://ftp.osuosl.org/pub/sous-chefs/bento/fedora-latest.box
-  - name: ubuntu-18.04
```

Comment:
you should not modify third party cookbooks, please restore these changes. They are not impacting our recipes.

Reply:
Sure.